src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1514,6 +1514,36 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                     <div>
                         <div data-test="PlotsTabPlotDiv" className="borderedChart posRelative">
                             <ScrollBar style={{position:'relative', top:-5}} getScrollEl={this.getScrollPane} />
+                            <div style={{textAlign:"center", position:"relative", top:-5, marginBottom:"-10px", minWidth: this.mutationDataCanBeShown && this.cnaDataCanBeShown ? 600 : 0}}>
```

Comment:
Changed style at here, and added a min-width just for the case when we show both checkboxes,